lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1986,7 +1986,7 @@ sub investigate {
     return {error => 'No result directory available for current job'} unless $self->result_dir();
     my $ignore = OpenQA::App->singleton->config->{global}->{job_investigate_ignore};
     for my $prev (@previous) {
-        if ($prev->result eq 'failed') {
+        if ($prev->result =~ /^failed$|incomplete/) {
```

Comment:
```suggestion
        if ($prev->result =~ /^(failed|incomplete)$|/) {
```

Although I would prefer using the constants `FAILED`/`INCOMPLETE`.

Reply:
yes, I tried that as well but all of that is not working: https://github.com/os-autoinst/openQA/pull/3981#discussion_r665682365

Reply:
Not sure why it is not working but I added another review comment for `should_show_autoinst_log`.